sample-apps/objc-sample-app/objc-sample-app/AppDelegate.m

```diff
@@ -33,8 +33,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     config.sandboxPushIntegrationName = @"objc-sample-app";
     
     [IterableAPI initializeWithApiKey: iterableApiKey
-                        launchOptions:launchOptions
-                               config:config];
+                        launchOptions: launchOptions
+                               config: config];
+    
+    IterableAPI.email = @"jay.kim@iterable.com";
```

Comment:
We should probably remove API Key and email from code and just add comments for users to add their apiKey and email.

Reply:
// IterableAPI.email = @"your.email@example.com";